src/pages/products/styles.css

```diff
@@ -13,27 +13,28 @@
 }
 
 .title {
-  display: flex;
-  justify-content: center;
+  margin: 0 0 0 280px;
 }
 
 .editableTitle,
 .editablePrice {
-  font: 25px/25px "MyriadProRegular";
+  font: 25px/30px "MyriadProRegular";
   background: none;
   width: 130px;
   height: 25px;
   border: 1px solid rgb(202, 202, 202);
   outline: none;
   color: rgb(202, 202, 202);
+  padding: 0 0 0 5px;
 }
```

Comment:
why do you keep styles for one component in another component?

Reply:
Because styles come from props and they are different. The difference in width

Reply:
i fix this